### PR TITLE
replace hex codes around the site with variables where possible

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Replace all instances of hex codes that are associated with color palette variables with the variable name.

NOTE: this PR is obviously massive. I think a skim is sufficient here.

## WHY
We want to start using the color variables rather than hex codes to make changes easier to propagate sitewide.

## HOW
Lots of global searching.

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
https://www.notion.so/quill/Replace-all-instances-of-green-hex-codes-on-website-with-variables-bdeeb0ca539f417783ce00903ffa64e5?pvs=4

https://www.notion.so/quill/Replace-all-instances-of-red-hex-codes-on-website-with-variables-ac1796fd005f45ed803cc908cc3c0a30?pvs=4

https://www.notion.so/quill/Replace-all-instances-of-blue-hex-codes-on-website-with-variables-5448fe7c0c8740c6ba6bf9e12ec8776b?pvs=4

https://www.notion.so/quill/Replace-all-instances-of-grey-hex-codes-on-website-with-variables-186589ae3dee43b3aed6759c5083fe11?pvs=4

https://www.notion.so/quill/Replace-all-instances-of-maroon-hex-codes-on-website-with-variables-65bd96f1b5b14f8ca60f7fcfb3fae1ac?pvs=4

https://www.notion.so/quill/Replace-all-instances-of-gold-hex-codes-on-website-with-variables-a8fdb0743129412fab97c36ab9903783?pvs=4

https://www.notion.so/quill/Replace-all-instances-of-teal-hex-codes-on-website-with-variables-ef9cb13bd085459f97847fa204071e26?pvs=4

https://www.notion.so/quill/Replace-all-instances-of-purple-hex-codes-on-website-with-variables-17768b4371524270bb070be1a19de11d?pvs=4

https://www.notion.so/quill/Replace-all-instances-of-violet-hex-codes-on-website-with-variables-c8d601e500744bf8a0222f5a763336ab?pvs=4

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? | NO - css changes
Have you deployed to Staging? | NO - non-app change
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | YES

### CODE REVIEW

**Review by brendanshean — APPROVED** [review]
Thanks for cleaning all of this up

File: services/QuillLMS/app/views/evidence/_first_topic.html.erb [hunk 1]
```diff
@@ -2,5 +2,5 @@
   <p class="topic">Culture & Society Topics</p>
   <img class="lazyload" alt="photograph of a football" loading="lazy" src="https://assets.quill.org/images/evidence/home_page/EvidenceLandingPage_Image_Football.webp" src="https://assets.quill.org/images/evidence/home_page/EvidenceLandingPage_Image_Football.webp"/>
   <p class="activity-title">"Should Schools Have Grade Requirements for Student Athletes?"</p>
-  <a href='https://www.quill.org/evidence/#/play?uid=180' target="_blank" class="q-button text-quillteal bg-white">View a sample activity</a>
+  <a href='https://www.quill.org/evidence/#/play?uid=180' target="_blank" class="q-button text-quill-teal bg-white">View a sample activity</a>
```
Comment (eadams17) [inline services/QuillLMS/app/views/evidence/_first_topic.html.erb:5]: I think this and the other instances of this class below need to be reverted or the class name updated from `text-quillteal` to `text-quill-teal`

**Review by eadams17 — APPROVED** [review]
Thanks for updating all of these! Just a comment about a potential breaking change with a class name, but otherwise looks good to me

File: services/QuillLMS/app/views/evidence/_first_topic.html.erb (hunk 1 above)
Comment (emilia-friedberg) [reply services/QuillLMS/app/views/evidence/_first_topic.html.erb:5]: @eadams17 thank you for flagging this! I think the class name has been updated here: https://github.com/empirical-org/Empirical-Core/blob/feat/replace-greens-with-variables-where-possible/services/QuillLMS/client/app/assets/styles/home.scss#L189 but let me know if I'm missing something.

Comment (eadams17) [reply services/QuillLMS/app/views/evidence/_first_topic.html.erb:5]: yeah, you got it! disregard-- it was just some hover states for buttons I was worried about, but looks like you got them covered here: https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/assets/styles/tool-section.scss#L24

**Pull request merged**